gradle.properties

```diff
@@ -1,9 +1,9 @@
-theForgeVersion=1.12.1-14.22.0.2460
-theMappingsVersion=snapshot_20170822
-malisisCoreMinVersion=1.12-6.0.0-SNAPSHOT
-malisisCoreVersion=1.12-6.0.5-SNAPSHOT
+theForgeVersion=1.11.2-13.20.1.2454
```

Comment:
Oh god what did you do?

Reply:
tried to update to the current branch, but that isn't a good idea mid work.
going to close this and make ANOTHER PR...